bowparquet.go

```diff
@@ -259,10 +257,84 @@ func (b *bow) WriteParquet(path string, verbose bool) error {
 	return nil
 }
 
-func validateColTypesMeta(b Bow, values string) (colTypesMeta []parquetColTypesMeta, err error) {
-	err = json.Unmarshal([]byte(values), &colTypesMeta)
+// GetParquetMetaColTimeUnit attempts to get the time unit of the column as a time.Duration
+// from the bow metadata read from a parquet file
+// If no time unit metadata is found, time.Duration(0) is returned without error.
```

Comment:
I already thought about it, I chose to return 0 because 0 is not a duration in real life and no error happened.
On the client side, it doesn't make any difference in my opinion.
What do you think?

Reply:
my opinion is that a client that actually use this method expect a actual result in a set of valid duration, returning 0 guessing it's not a duration force the user to be extra careful about the result. For instance if I use this to initialize a time standardisation in nanosec and do value * timeUnit I risk to do a value * 0 that would not break the code until displaying or (hopefully it would break) storing results. If I get an error, I don't take risk and if as a user I know it can be empty I explicitly says `if errors.Is(err, bow.ErrTimeUnitNotFound) {timeUnit = defaultTimeUnit}` -> explicit code for users defined convention.

Reply:
Ok! I get your point thanks :)